assets/js/modules/search-console/components/dashboard/GoogleSitekitSearchConsoleDashboardWidget.js

```diff
@@ -58,13 +60,21 @@ const GoogleSitekitSearchConsoleDashboardWidget = () => {
 	const [ loading, setLoading ] = useState( true );
 	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
 	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
+	const isDomainProperty = useSelect( ( select ) => select( STORE_NAME ).isDomainProperty() );
+	const referenceSiteURL = useSelect( ( select ) => {
+		return unTrailingSlashIt( select( CORE_SITE ).getReferenceSiteURL() );
+	} );
```

Comment:
This can be in a single line like other simple `useSelect` usages.

Reply:
@tofumatt recommended this style in favor of readability. [see comment](https://github.com/google/site-kit-wp/pull/2229#discussion_r512850815)